std/selector/multiplexer.go

```diff
@@ -43,7 +42,7 @@ func Map(api frontend.API, queryKey frontend.Variable,
 	// we don't need this check, but we added it to produce more informative errors
 	// and disallow len(keys) < len(values) which is supported by generateSelector.
 	if len(keys) != len(values) {
-		panic("The number of keys and values must be equal")
+		log.Panicf("The number of keys and values must be equal (%d != %d)", len(keys), len(values))
```

Comment:
Please revert. Imo log.Panicf writes to standard logger. gnark already has its "standard" logger and this makes configuring the output difficult. See https://github.com/ConsenSys/gnark/blob/master/logger/logger.go

Reply:
Oh, sorry I didn't know that.